govc/vm/info.go

```diff
@@ -100,7 +102,16 @@ func (cmd *info) Run(f *flag.FlagSet) error {
 				continue
 			}
 
-			res.VirtualMachines = append(res.VirtualMachines, mvm)
+			host, err := vm.HostSystem(ctx)
```

Comment:
This will do an extra call to find the host system through the summary property of the virtual machine managed object. I think it is possible to extract this from the properties that already retrieved in the `RetrieveOne` call. Then all that's needed is a `object.NewHostSystem` to get the corresponding object.